### PR TITLE
Navbar footer

### PR DESCRIPTION
**Pull request by tarunsankhla**
No new changes done just drag and drop of html and css and added backend ans server js fo mock bee

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Code is readable as of now.

**Review by m26unkwn — APPROVED** [review]

**Comment by PuneetP16** [comment]
Conversion of html to JSX looks good. Good to merge.

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+import { Navbar } from './components/Navbar/Navbar';
+import { Footer } from './components/Footer/Footer';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+      {/* <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
```
Comment (PuneetP16) [inline src/App.js]: Commented code could be removed.

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+import { Footer } from './components/Footer/Footer';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+      {/* <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" /> 
+
```
Comment (PuneetP16) [inline src/App.js]: Images could be converted into webp for better performance

File: src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    display: flex;
+    justify-content: space-around;
+    font-size: 2rem;
+    flex-wrap: wrap;
+    flex-direction: row;
+    padding-top: 1rem;
+}
+
+/* .navbar .project-title{
+    margin: 1rem;
```
Comment (PuneetP16) [inline src/components/Navbar/Navbar.css]: Commented unused code could be removed

**Review by PuneetP16 — APPROVED** [review]

File: src/components/Navbar/Navbar.css (hunk 3 above)
Comment (tarunsankhla) [reply src/components/Navbar/Navbar.css]: done

**Pull request merged**